cwms_radar_standalone/src/main/java/mil/army/usace/hec/RadarAPI.java

```diff
@@ -77,7 +78,7 @@ public static void main(String[] args){
     public RadarAPI(javax.sql.DataSource ds, int port){
         this.port = port;
         PolicyFactory sanitizer = new HtmlPolicyBuilder().disallowElements("<script>").toFactory();
-        JavalinValidation.register(UnitSystem.class, v -> UnitSystem.systemFor(v) );
+        JavalinValidation.register(UnitSystem.class, UnitSystem::systemFor);
```

Comment:
Excellent cleanup. I need to stop forgetting about this particular syntax.

Reply:
I mostly changed it just so sonarqube stops reminding me about it.  I'm still getting used to the method reference syntax - I wouldn't say its my go-to pattern.